browser/android/tab_features_android.cc

```diff
@@ -21,6 +24,13 @@ TabFeaturesAndroid::TabFeaturesAndroid(content::WebContents* web_contents,
         TabAndroid::FromWebContents(web_contents)->GetAndroidId(),
         web_contents);
   }
+
+  if (base::FeatureList::IsEnabled(
+          youtube_script_injector::features::kBraveYouTubeScriptInjector)) {
+    youtube_script_injector_ =
```

Comment:
this is correct, but you're instantiating it directly here which is not https://github.com/brave/brave-core/pull/26355/files#r2034066008